.github/workflows/deploy-dev.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Yarn build
         # Set environment variables required to perform the build. These are only available to this step
         env:
-          VITE_VERSION: ${{ github.sha }}
+          VITE_VERSION: ${{ github.event.client_payload.tag }}
```

Comment:
No, here it should be the sha with which it was called. `event.client_payload.tag` is the custom payload that deploy send when we push a stack. It is not standard.